coffee_roaster_market/apps/coffee/tests/test_serializers.py

```diff
@@ -0,0 +1,118 @@
+import pytest
+
+from coffee_roaster_market.apps.coffee.models import (
+    SensorialProfileModel,
+    GeoLocationModel,
+    PlantationModel,
+    CoffeeModel,
+)
+from coffee_roaster_market.apps.coffee.serializers import (
+    SensorialProfilSerializer,
+    GeoLocationSerializer,
+    PlantationSerializer,
+    CoffeeSerializer,
+)
+
+from .factories import GeolocationFactory, PlantationFactory, SensorialProfileFactory
+from .assets import parse_json_data
+
+pytestmark = pytest.mark.django_db
+
+
+def test_sensorial_profile_serializer():
+    expected_overall = 10
+    data = b"""{
+        "accidity": 10,
+        "balance": 10,
+        "body": 10,
+        "flavor": 10,
+        "aftertaste": 10,
+        "sweetness": 10,
+        "clean_cup": 10
+    }"""
+    data_dict = parse_json_data(data)
+    serializer = SensorialProfilSerializer(data=data_dict)
```

Comment:
`SensorialProfil(e)Serializer` 🤔 ?? Is a typo?

Reply:
Yes, thanks 😄